README.md

```diff
@@ -1,41 +1,56 @@
-# HTML template for the AsyncAPI Generator
+<h5 align="center">
+  <br>
+  <a href="https://www.asyncapi.org"><img src="https://github.com/asyncapi/parser-nodejs/raw/master/assets/logo.png" alt="AsyncAPI logo" width="200"></a>
+  <br>
+  HTML template for the AsyncAPI Generator
+</h5>
+
+HTML template for the [AsyncAPI Generator](https://github.com/asyncapi/generator) using an [AsyncAPI React Component](https://github.com/asyncapi/asyncapi-react) under the hood.
+
+---
+
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
```

Comment:
would be great to have weekly or monthly downloads badge 😄

Reply:
I wrote year when Kyma give AsyncAPI 100% control of project :) I don't know if we need to clarify it.

Reply:
I added monthly :)